sys/include/net/ng_netdev.h

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_netdev Network device driver interface
+ * @addtogroup  net
+ * @{
+ *
+ * @file
+ * @brief       Network device driver interface definition.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef __NETDEV_H_
+#define __NETDEV_H_
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "net/ng_pktsnip.h"
+#include "net/ng_netconf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Type for @ref msg_t if device fired an event
+ */
+#define NETDEV_MSG_EVENT_TYPE           (0x0100)
```

Comment:
really? I feel that at some point we start to surround every single define with '#ifndef'... I would rather leave at is and adjust the chosen value if needed -> wasn't there some discussion for system wide message numbers anyway?